### DISCUSSION TITLE
Disconnect college email from a lost github account

### DISCUSSION THREAD

**Discussion by psychoSherlock** [community/community#131948] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Github,
I am writing on behalf of a classmate. She had a github account before and was signing up for co-pilot. On which she was asked to setup a TOTP. She was new to TOTP so she downloaded a useless app instead of Google or Microsoft Authenticator. Later she was given access to copilot after verifying her college email and id card.

Later she deleted that app which left it without any backup. And because she had no Github Mobile she lost complete access to her account. She never even used github ssh. She was very new.

I know that github wont give us access back to the account. But it would be great if you could disconnect her college email so that she can use it on new github account and access co pilot.  She is just a first year and it would be very usefull for her career.
Copilot helps us a lot...

We are ready to mail you and verify our emails and college.

**Comment by paiineli** [comment]
Hi,

Here are some recommendations you can follow to try to resolve the issue:

Contact GitHub Support: He should send an email detailing the situation to GitHub Support, including information about the account, the college email, and any documentation proving he is a student.

Identity Verification: If necessary, he can offer to send a copy of the college ID or other proof to verify his identity.

Create a New Email: If support cannot disconnect the college email, he might consider creating a new email address (if possible) and using that to create a new GitHub account.

Use Reliable Authenticators: Once he gains access to a new account, it’s important to use a reliable authenticator app, like Google Authenticator or Microsoft Authenticator, and back up the keys.

Document the Process: He should keep a record of all interactions with GitHub Support and any steps he takes for future reference.

Join Developer Communities: He can seek help in forums or developer communities, like Stack Overflow, where others may have faced similar issues.

These actions can help resolve the problem and allow her to regain access to GitHub and Copilot.

**Comment by ebndev** [comment, marked as answer]
Hi @psychoSherlock,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
